packages/firebase_remote_config/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_remote_config;
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig extends FirebasePluginPlatform with ChangeNotifier {
+  // Cached instances of [FirebaseRemoteConfig].
+  static final Map<String, RemoteConfig> _firebaseRemoteConfigInstances = {};
+
+  // Cached and lazily loaded instance of [FirebaseRemoteConfigPlatform]
+  // to avoid creating a [MethodChannelFirebaseRemoteConfig] when not needed
+  // or creating an instance with the default app before a user specifies an
+  // app.
+  FirebaseRemoteConfigPlatform _delegatePackingProperty;
+
+  /// Returns the underlying delegate implementation.
+  ///
+  /// If called and no [_delegatePackingProperty] exists, it will first be
+  /// created and assigned before returning the delegate.
+  FirebaseRemoteConfigPlatform get _delegate {
+    if (_delegatePackingProperty == null) {
```

Comment:
We could use `_delegatePackingProperty ??=`

Reply:
fancy! done.